### ISSUE TITLE
[New Rule] HAFNIUM MS Exchange UM Service Writing - CVE-2021-26858 (2021 - March)

### ISSUE THREAD

**Issue by austinsonger** [elastic/detection-rules#973]
## Description

Microsoft Exchange Server’s Unified Messaging service creating non-standard content on disk, which could indicate web shells or other malicious content. CVE-2021-26858

### Target indexes

filebeat-*

### Target Operating Systems

Windows

### Platforms

Microsoft Exchange Server

### Query

```
(winlog.channel : \"Security\" and winlog.event_id : \"4663\" and process.executable.text : (*\\UMWorkerProcess.exe or *\\UMService.exe) and winlog.event_data.ObjectName : (*.php or *.jsp or *.js or *.aspx or *.asmx or *.asax or *.cfm or *.shtml) and winlog.event_data.AccessMask : (\"0x2\" or \"0x100\" or \"0x10\" or \"0x4\"))
```

### References

- https://www.volexity.com/blog/2021/03/02/active-exploitation-of-microsoft-exchange-zero-day-vulnerabilities/
- https://www.microsoft.com/security/blog/2021/03/02/hafnium-targeting-exchange-servers/

## Example Data

```json
{
   "author": [
      "Austin Songer"
   ],
   "actions": [],
   "created_at": "2021-03-04T04:45:30.083Z",
   "updated_at": "2021-03-04T05:35:02.483Z",
   "created_by": "asonger",
   "description": "Exchange server UM process writing suspicious files that may be indicative of webshells.",
   "enabled": true,
   "false_positives": [
      "unknown"
   ],
   "filters": [],
   "from": "now-360s",
   "id": "71ee1040-7ca4-11eb-93c4-73664be03f52",
   "immutable": false,
   "index": [
      "filebeat-*",
      "logs-*",
      "winlogbeat-*"
   ],
   "interval": "5m",
   "rule_id": "9374e81a-d9ad-4603-a641-76889cb8fcff",
   "language": "kuery",
   "license": "",
   "output_index": ".siem-signals-default",
   "max_signals": 100,
   "risk_score": 47,
   "risk_score_mapping": [],
   "name": "Microsoft Exchange Server UM Service Writing Vulnerability CVE-2021-26858",
   "query": "(winlog.channel : \\\"Security\\\" and winlog.event_id : \\\"4663\\\" and process.executable.text : (*\\\\UMWorkerProcess.exe or *\\\\UMService.exe) and winlog.event_data.ObjectName : (*.php or *.jsp or *.js or *.aspx or *.asmx or *.asax or *.cfm or *.shtml) and winlog.event_data.AccessMask : (\\\"0x2\\\" or \\\"0x100\\\" or \\\"0x10\\\" or \\\"0x4\\\"))",
   "references": [
      "https://www.volexity.com/blog/2021/03/02/active-exploitation-of-microsoft-exchange-zero-day-vulnerabilities/"
   ],
   "meta": {
      "from": "1m",
      ""
   },
   "severity": "medium",
   "severity_mapping": [],
   "updated_by": "asonger",
   "tags": [
      "Elastic",
      "Windows",
      "T1190",
      "Initial Access",
      "TA0001",
      "Exploit Public-Facing Application"
   ],
   "to": "now",
   "type": "query",
   "threat": [
      {
         "framework": "MITRE ATT&CK",
         "tactic": {
            "id": "TA0001",
            "reference": "https://attack.mitre.org/tactics/TA0001",
            "name": "Initial Access"
         },
         "technique": [
            {
               "id": "T1190",
               "reference": "https://attack.mitre.org/techniques/T1190",
               "name": "Exploit Public-Facing Application",
               "subtechnique": []
            }
         ]
      }
   ],
   "throttle": "no_actions",
   "version": 2,
   "exceptions_list": []
}

```

**Comment by peasead** [comment]
Hey, @austinsonger thanks for opening this rule (and #972).

Do you plan on making a PR? We're more than happy to, but didn't want to swoop in if you had something in the mix.

**Comment by austinsonger** [comment]
Would you folks prefer that I just create a pull request for new rules instead of creating a issue. It just seems like every new rule has a corresponding issue. Or would you just prefer that when I have the `.toml.` file complete, to just do the pull request and create the issue and link the PR?

**Comment by threat-punter** [comment]
Hi @austinsonger. The preferred workflow is to open an issue first and then link your PR to the issue using GitHub's closing keywords like "Resolves #1".
